web-registry/src/components/templates/ProjectDetails/ProjectDetails.tsx

```diff
@@ -225,6 +225,8 @@ function ProjectDetails(): JSX.Element {
             ? () => setIsBuyFlowStarted(true)
             : () => setDisplayErrorBanner(true)
         }
+        onChainProjectId={onChainProjectId as string}
```

Comment:
best to avoid the type assertion and just use the initial `onChainProjectId` type in `SellOrdersActionsBar` props, this can be null

Reply:
the initial type is kind of confusing to find.

the error without `onChainProjectId as string` was:
```
[E] Type 'Maybe<string> | undefined' is not assignable to type 'string | undefined'. Type 'null' is not assignable to type 'string | undefined'.
```

in retrospect I guess the fix is quite obvious because of the last sentence in the message above. but when asking for the type info for `onChainProjectId` it's showing this:
```
const onChainProjectId: Maybe<string> | undefined
```

which leads you to wonder, what is `Maybe`? and that's why i said it's hard to find, because it's buried in `web-registry/src/generated/graphql.tsx:3`:
```
export type Maybe<T> = T | null;
```

should i be importing that type and using that?

here's how i've done it for now: 80335fcf

Reply:
yes (it's already done multiple times in the app if you look for it) or you can simply use `.. | null` since it's the same